src/code/z_viscvg.c

```diff
@@ -75,20 +109,29 @@ void func_800ACE98(struct_801664F0* this, Gfx** gfxp) {
     }
 
     switch (this->type) {
-        case 1:
-            gSPDisplayList(gfx++, D_8012AC40);
+        case FB_FILTER_CVG_RGB:
+            gSPDisplayList(gfx++, sCoverageRGBDL);
             break;
-        case 2:
-            gDPSetColor(gfx++, G_SETPRIMCOLOR, this->color.rgba);
-            gSPDisplayList(gfx++, D_8012AC58);
+
+        case FB_FILTER_CVG_RGB_PRIMFOG:
+            // Set primitive color for uniform fogging in custom RenderMode
+            gDPSetColor(gfx++, G_SETPRIMCOLOR, this->primColor.rgba);
+            gSPDisplayList(gfx++, sCoverageRGBPrimFogDL);
             break;
-        case 3:
-            gDPSetColor(gfx++, G_SETBLENDCOLOR, this->color.rgba);
-            gSPDisplayList(gfx++, D_8012AC00);
+
+        case FB_FILTER_CVG:
+            // Set background color for G_RM_VISCVG
+            gDPSetColor(gfx++, G_SETBLENDCOLOR, this->primColor.rgba);
+            gSPDisplayList(gfx++, sCoverageOnlyDL);
+            break;
+
+        case FB_FILTER_CVG_RGB_FOG:
+            // Set fogging color for custom RenderMode, needs to be close to 0 to not overflow
```

Comment:
```suggestion
            // Set fog color for custom RenderMode, needs to be close to 0 to not overflow
```
the name of the color register is "fog color", typically used for the fog effect but not necessarily, so for refering to it it's probably best to stick to its name